### PR TITLE
Skios 151 latex roadmap chapter

### PR DESCRIPTION
**Pull request by grossamos**
# Skiosa - PR
-  x] documentation can't be split up into smaller logical units
- [ ] PRE-MERGE add tag to start drone automation

### CODE REVIEW

File: content/03kapitel.tex [hunk 1]
```diff
@@ -2,6 +2,51 @@
...
+    \label{fig:roadmap}
+\end{figure}
+
+\subsection{Initial Infrastructure}
+Ziel des \ac{PI}s ``Initial Infrastructure''\footnote{dt. Anfängliche Infrastruktur} ist es die Grundstruktur für die Plattform aufzusetzen und Workflow Elemente wie eine \ac{CI/CD} Pipeline, das Kubernetes Cluster und die initiale Servicestruktur zu erstellen.
+Fragen rund um, wie die Service-Architektur und Infrastruktur aufzusetzen ist, gilt hierbei zu klären.
+Am Ende dieses \ac{PI}s soll das Ergebnis eine funktionsfähige Infrastruktur sein, auf welcher in kommenden Sprints entwickelt und Sprint-Ergebnisse ausgeliefert werden können.
+
+\subsection{Article Views and Base Homepage}
+Ziel des \ac{PI}s ``Article Views and Base Homepage''\footnote{dt. Homepage und Einsichtseiten} ist es die wichtigsten Seiten der Applikation zu erstellen.
```
Comment (jeppard) [inline content/03kapitel.tex:31]:
```suggestion
Ziel des \acp{PI} \enqoute{Article Views and Base Homepage}\footnote{dt. Artikel-Seiten und Startseite} ist es die wichtigsten Seiten der Applikation zu erstellen.
```
* [ ] Replace all Quotes with \enqoute{}

File: content/03kapitel.tex [hunk 2]
```diff
@@ -2,6 +2,51 @@
 
 \chapter{Roadmap}
 
-\section{Epic 1}
+\section{Überblick}
+Nachdem die Requirements, Rollenaufteilung und Organisation des Projektes geklärt sind, geht es darum zu klären, was später in den tatsächlichen Sprints geschehen soll.
+Als Orientierungshilfe werden hierbei die Requirements des Projektes geordnet und in Product Increments aufgeteilt.
```
Comment (jeppard) [inline content/03kapitel.tex:7]:
```suggestion
Als Orientierungshilfe werden hierbei die Anforderungen des Projektes geordnet und in Product Increments aufgeteilt.
```

File: content/03kapitel.tex [hunk 3]
```diff
@@ -2,6 +2,51 @@
...
 
-\section{Epic 2}
+In diesem Kontext können \ac{PI}s als die geplanten größeren Versionssprünge unserer Plattform verstanden werden.
+Wichtig ist es hierbei hervorzuheben, dass \ac{PI}s nicht die einzelnen Sprintziele darstellen.
```
Comment (jeppard) [inline content/03kapitel.tex:10]:
```suggestion
Hierbei ist es wichtig hervorzuheben, dass \ac{PI}s nicht die einzelnen Sprintziele darstellen.
```

File: content/03kapitel.tex [hunk 4]
```diff
@@ -2,6 +2,51 @@
...
+Diese im Voraus (mit festen zeitlichen Markern) festzulegen hätte wenig mit der im Team erzielten agilen Projektorganisation zu tun.
+Vielmehr stellen die \ac{PI}s des Projektes den geplanten Pfad dar, in welchem die Requirements abgearbeitet werden sollen.
+
+\section{Product Increments}
+Aus der obig erläuterten Überlegung leitet sich sie in Abbildung \ref{fig:roadmap} dargestellte Roadmap ab.
```
Comment (jeppard) [inline content/03kapitel.tex:15]: Ist zu dem "obig" eine Referenz möglich? Welches Kapitel?

File: content/03kapitel.tex [hunk 5]
```diff
@@ -2,6 +2,51 @@
...
+\begin{figure}
+    \centering
+    \includegraphics[width=0.3\linewidth]{roadmap.png}
+    \caption{Roadmap des Projektes}
+    \label{fig:roadmap}
+\end{figure}
+
+\subsection{Initial Infrastructure}
+Ziel des \ac{PI}s ``Initial Infrastructure''\footnote{dt. Anfängliche Infrastruktur} ist es die Grundstruktur für die Plattform aufzusetzen und Workflow Elemente wie eine \ac{CI/CD} Pipeline, das Kubernetes Cluster und die initiale Servicestruktur zu erstellen.
+Fragen rund um, wie die Service-Architektur und Infrastruktur aufzusetzen ist, gilt hierbei zu klären.
```
Comment (jeppard) [inline content/03kapitel.tex:27]:
```suggestion
```
Ich habe den Satz nicht verstanden :/ Für mich hat er das gleiche ausgesagt wie der Satz oben drüber, und war umständlich formuliert

File: content/03kapitel.tex [hunk 6]
```diff
@@ -2,6 +2,51 @@
...
+Am Ende dieses \ac{PI}s soll das Ergebnis eine funktionsfähige Infrastruktur sein, auf welcher in kommenden Sprints entwickelt und Sprint-Ergebnisse ausgeliefert werden können.
+
+\subsection{Article Views and Base Homepage}
+Ziel des \ac{PI}s ``Article Views and Base Homepage''\footnote{dt. Homepage und Einsichtseiten} ist es die wichtigsten Seiten der Applikation zu erstellen.
+Auf Basis von Mockdaten sollen hierbei die Requirements in Richtung der Einsichtseiten von Feed und Artikel Einsichtseiten bzw. der Homepage mit vorgeschlagenen Artikeln erfüllt werden.
```
Comment (jeppard) [inline content/03kapitel.tex:32]:
```suggestion
Auf Basis von Beispieldaten sollen hierbei die Anforderungen in Richtung der Seiten der Feeds und Artikel bzw. der Startseite mit vorgeschlagenen Artikeln erfüllt werden.
```

File: content/03kapitel.tex [hunk 7]
```diff
@@ -2,6 +2,51 @@
...
+
+
+\subsection{User Interaction}
+Ziel des \ac{PI}s ``User Interaction''\footnote{dt. Bernutzerinteraktion} soll das Ausbauen des User-Management Requirements sein.
```
Comment (jeppard) [inline content/03kapitel.tex:36]:
```suggestion
Ziel des \ac{PI}s ``User Interaction''\footnote{dt. Benutzerinteraktion} soll das Ausbauen des User-Management Anforderung sein.
```

File: content/03kapitel.tex [hunk 8]
```diff
@@ -2,6 +2,51 @@
...
+Hierbei sollen sich Benutzer in die Plattform einloggen können, Feeds zur Plattform hinzufügen und abonnieren können bzw. Artikel liken und speichern können.
+Da es nun einen Weg gibt Artikel zur Plattform hinzuzufügen, ist es ebenfalls Teil dieses \ac{PI}s statt Mockdaten echte RSS Feeds zu verwenden.
```
Comment (jeppard) [inline content/03kapitel.tex:38]:
```suggestion
Da es nun einen Weg gibt Artikel zur Plattform hinzuzufügen, ist es ebenfalls Teil dieses \ac{PI}s, statt Beispieldaten echte RSS Feeds zu verwenden.
```

File: content/03kapitel.tex [hunk 9]
```diff
@@ -2,6 +2,51 @@
...
+
+\subsection{Recommendations}
+Ziel des \ac{PI}s ``Recommendations''\footnote{dt. Vorschläge} ist es die Recommendations der Plattform auszubauen.
```
Comment (jeppard) [inline content/03kapitel.tex:41]:
```suggestion
Ziel des \ac{PI}s ``Recommendations''\footnote{dt. Vorschläge} ist es die Vorschläge der Plattform auszubauen.
```

File: content/03kapitel.tex [hunk 10]
```diff
@@ -2,6 +2,51 @@
...
+Requirements wie das Angeben von Präferenzen in Richtung Kategorien und vorgeschlagenen Feeds gilt es hier zu implementieren.
```
Comment (jeppard) [inline content/03kapitel.tex:42]:
```suggestion
Anforderungen wie das Angeben von Präferenzen mithiolfe von Kategorien und vorgeschlagenen Feeds gilt es hier zu implementieren.
```

File: content/03kapitel.tex [hunk 11]
```diff
@@ -2,6 +2,51 @@
...
+Am Ende des \ac{PI}s wäre eine deutliche Verbesserung der Vorschläge für Nutzer zu erkennen.
+
+\subsection{Enhanced Usability}
+Ziel des \ac{PI}s ``Enhanced Usability''\footnote{dt. Verbesserte Benutzbarkeit} ist es mögliche weitere Elemente der als unrealistisch eingeschätzten Requirements zu implementieren.
```
Comment (jeppard) [inline content/03kapitel.tex:46]:
```suggestion
Ziel des \ac{PI}s ``Enhanced Usability''\footnote{dt. Verbesserte Benutzbarkeit} ist es mögliche weitere Elemente der als unrealistisch eingeschätzten Anforderungen zu implementieren.
```

File: content/03kapitel.tex [hunk 12]
```diff
@@ -2,6 +2,51 @@
...
+Größter Fokus dieses Product Increments ist es ein Benachrichtigungssystem für Benutzer, sowie die Content-Creator Page zu implementieren.
```
Comment (jeppard) [inline content/03kapitel.tex:47]:
```suggestion
Größter Fokus dieses \ac{PI} ist es ein Benachrichtigungssystem für Benutzer, sowie die Content-Creator Page zu implementieren.
```

**Review by jeppard — CHANGES_REQUESTED** [review]

**Comment by jeppard** [comment]
Not all changes are compulsory. But the technical changes with the checkboxes are compulsory.

File: content/03kapitel.tex [hunk 13]
```diff
@@ -2,6 +2,51 @@
 
 \chapter{Roadmap}
 
-\section{Epic 1}
+\section{Überblick}
+Nachdem die Requirements, Rollenaufteilung und Organisation des Projektes geklärt sind, geht es darum zu klären, was später in den tatsächlichen Sprints geschehen soll.
```
[the remark this answers is not in the document]
Comment (grossamos) [reply content/03kapitel.tex:6]: added

File: content/03kapitel.tex [hunk 14]
```diff
@@ -2,6 +2,51 @@
...
+Als Orientierungshilfe werden hierbei die Requirements des Projektes geordnet und in Product Increments aufgeteilt.
 
-\section{Epic 2}
+In diesem Kontext können \ac{PI}s als die geplanten größeren Versionssprünge unserer Plattform verstanden werden.
```
[the remark this answers is not in the document]
Comment (grossamos) [reply content/03kapitel.tex:9]: fixed

File: content/03kapitel.tex (hunk 1 above)
Comment (grossamos) [reply content/03kapitel.tex:31]: fixed, 
image
still looks like we're using english quotes though

File: content/03kapitel.tex (hunk 2 above)
Comment (grossamos) [reply content/03kapitel.tex:7]: Added (resentfully)

File: content/03kapitel.tex (hunk 3 above)
Comment (grossamos) [reply content/03kapitel.tex:10]: added

File: content/03kapitel.tex (hunk 8 above)
Comment (grossamos) [reply content/03kapitel.tex:38]: added

File: content/03kapitel.tex (hunk 9 above)
Comment (grossamos) [reply content/03kapitel.tex:41]: added

File: content/03kapitel.tex (hunk 10 above)
Comment (grossamos) [reply content/03kapitel.tex:42]: fixed typeo in your suggestion and added it

File: content/03kapitel.tex (hunk 11 above)
Comment (grossamos) [reply content/03kapitel.tex:46]: added (resentfully)

File: content/03kapitel.tex (hunk 12 above)
Comment (grossamos) [reply content/03kapitel.tex:47]: Replaced

**Comment by grossamos** [comment]
Changes acted upon :+1:

File: content/03kapitel.tex (hunk 1 above)
Comment (grossamos) [reply content/03kapitel.tex]: Never mind, forgot to merge your changes

**Review by jeppard — APPROVED** [review]
Thanks looks good to me. Whats pur worklflow Do we need a second reviewer? Otherwise create Tag and merge.

**Comment by grossamos** [comment]
Since I'm the "service" lead for documentation, no second reviewer is needed, will merge

**Pull request merged**